tests/com/rcv/TabulatorTests.java

```diff
@@ -89,33 +90,36 @@ static boolean fileCompare(String path1, String path2) {
     return result;
   }
 
+  // function: getConfigPath
+  // purpose: given stem returns path to config file in test asset folder
+  // returns: path to config file
+  static String getConfigPath(String stem) {
```

Comment:
You could generalize the `getConfigPath` function to generate the `expectedPath` too (e.g. with a simple string parameter that inserts whichever word after the underscore and before .json). You should rename the function if you do this though.

Reply:
dig it